Makefile

```diff
@@ -84,6 +84,7 @@ localkube-image: out/localkube
 	make -C deploy/docker VERSION=$(VERSION)
 
 minikube_iso:
+	echo $(ISO_VERSION) > deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/etc/VERSION
```

Comment:
Would you mind .gitignoring it? This will mean we'll end up with a dirty tree state during a normal iso build.